src/explorer/src/components/Footer.astro

```diff
@@ -13,24 +19,49 @@
         </svg>
       </div>
       <div class="pl-4">
-        <div class="text-lg font-semibold text-white">
-          An official website of the{' '}
+        <div class="text-lg font-semibold text-zinc-50">
+          An official website of the{" "}
           <span class="whitespace-no-wrap">State of Utah</span>
         </div>
-        <div class="text-sm text-white">©2023 State of Utah</div>
+        <div class="text-sm text-zinc-50">© State of Utah</div>
       </div>
     </div>
-    <div class="pt-6 text-center text-sm text-white lg:pt-0">
-      <div class="space-x-2 whitespace-pre-wrap lg:space-x-4">
-        <a href="https://www.utah.gov" target="__blank" rel="noreferrer"> Utah.gov Home</a>
-        <span class="border-l-2" aria-hidden></span>
-        <a href="https://www.utah.gov/disclaimer.html" target="__blank" rel="noreferrer"> Terms of Use</a>
-        <span class="border-l-2" aria-hidden></span>
-        <a href="https://www.utah.gov/privacypolicy.html" target="__blank" rel="noreferrer"> Privacy Policy</a>
-        <span class="border-l-2" aria-hidden></span>
-        <a href="https://www.utah.gov/accessibility.html" target="__blank" rel="noreferrer"> Accessibility</a>
-        <span class="border-l-2" aria-hidden></span>
-        <a href="https://www.utah.gov/translate.html" target="__blank" rel="noreferrer"> Translate</a>
+    <div class="pt-6 text-center text-sm lg:pt-0">
+      <div class="space-x-2 leading-10 text-zinc-50 lg:space-x-4">
+        <a
+          target="__blank"
+          rel="noreferrer"
+          href="https://www.utah.gov"
+          type="custom">Utah.gov Home</a
```

Comment:
**question**: can we safely remove the custom link property? We aren't applying default link styles in this project that need the custom type to opt out of right?

Reply:
Yes, great catch.